pkg/flash/chips/chips.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package chips contains chips known to work with the flash tool.
+package chips
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/u-root/u-root/pkg/flash/op"
+)
+
+const (
+	k = 1024
+	m = 1024 * 1024
+)
+
+type EraseBlock struct {
+	Size int
+	Op   uint8
+}
+
+type Chip struct {
+	Vendor      string
+	Chip        string
+	ID          int
```

Comment:
Can this be a dedicated type, `type ID int`?

Reply:
done.